### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by peschkaj**

### CODE REVIEW

**Comment by connorkuehl** [comment]
I read this. I am not a lawyer. It looks good to me. I would sign this.

What do others think?

**Comment by peschkaj** [comment]
I too would sign this, but I also generated it.

**Comment by TsebMagi** [comment]
Looks pretty good to me, I'd have your legal guy check it out.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+You must sign a Contribution License Agreement (CLA) before your PR will be merged. This a one-time requirement for Microsoft projects in GitHub. You can read more about Contribution License Agreements (CLA) on Wikipedia.
```
Comment (Zonr0) [inline CONTRIBUTING.md]: Microsoft projects on Github? Copy pasta mistake?

Comment (peschkaj) [inline CONTRIBUTING.md]: derp!

**Pull request merged**